.github/workflows/goreleaser.yml

```diff
@@ -18,10 +18,20 @@ jobs:
         run: echo "GORELEASER_CURRENT_TAG=${{ github.event.release.tag_name }}" >> $GITHUB_ENV
       - uses: actions/setup-go@0aaccfd150d50ccaeb58ebd88d36e91967a5f35b #v5.4.0
         with:
-          go-version-file: 'go.mod'
+          go-version-file: "go.mod"
       - name: Create .release-env file
         run: |-
           echo 'GITHUB_TOKEN=${{secrets.GORELEASER_ACCESS_TOKEN}}' >> .release-env
+          # extract the chain name from the tag
+          TAG_NAME="${{ github.event.release.tag_name }}"
+          if [[ "$TAG_NAME" == *"-mocha"* ]]; then
+            echo 'CHAIN_NAME=mocha' >> .release-env
+          elif [[ "$TAG_NAME" == *"-arabica"* ]]; then
+            echo 'CHAIN_NAME=arabica' >> .release-env
+          else
```

Comment:
[note]
this will treat `-rc` releases also as mainnet releases. is this something we want?

Reply:
I'd rather not want that whole goreleaser logic tbh, but rather https://github.com/celestiaorg/celestia-app/issues/4549#issuecomment-2810695347